src/pages/components/pagination/code.mdx

```diff
@@ -1,23 +1,37 @@
 ---
 title: Pagination
 description: Pagination is used for splitting up content or data into several pages, with a control for navigating to the next or previous page.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
 <ComponentDemo
```

Comment:
Theming bug "items per page" not changing color.

![image](https://user-images.githubusercontent.com/11670886/78407364-34ecd000-75ca-11ea-886b-d0de55d887f2.png)